UK_Reports/models/mail_template.py

```diff
@@ -30,3 +30,13 @@ class MailTemplate(models.Model):
 	def update_mail_templates(self):
 		self.env.ref('account.email_template_edi_invoice').write(dict(
 			report_template=self.env.ref('UK_Reports.uk_invoice_print').id))
+
+	#Adding Remittance advice mail template in the action
+	@api.model
+	def create(self, vals):
+		if 'name' in vals and vals['name'] == 'Remittance Advice':
```

Comment:
Usually we would do this with a xml function call. If possible could this be removed and changed to an xml function which calls `create_action()` on the external id `UK_Reports.email_template_remittance_advice_id` rather than a persistent hook in create which is basing the hook off a user changeable string?

Reply:
i.e:
```xml:
<odoo>
    <function
        model="mail.template"
        name="create_email_actions"
        id="create_email_actions"
    />
</odoo>
```
```python
@api.model
def create_email_actions(self):
    email_ext_ids = ["UK_Reports.email_template_remittance_advice_id"]
    [self.env.ref(ext_id).create_action() for ext_id in email_ext_ids
```

Reply:
After discussion, no change required due to the above not being applicable for this scenario, and the other suggested change (having a flag on the xml email template, and creating context action on create if that flag is set) being too much work for such little return.